index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="wrapper">
+        <div class="nav-bar">
+            <div class="logo">Charles</div>
+            <img src="images/Union.png" alt="Menu Bar">
+        </div>
+        <div class="content">
+            <article>
+                <header>
+                    <h2>I'm Charles Nice to meet you!</h2>
+
+                </header>
+                <p>
+                    My Name is Okoye Charles and I’m a software developer!
+                     I can help you build a product ,
+                    feature or website Look through some of my work and experience! 
+                    If you like what you see and need help on a project, I am available
+                     for hire.
+                </p>
```

Comment:
- [ ] for the title and paragraphs try to use the right font `font-family: 'Poppins';` as in the Figma design

Reply:
done